From 3e77e6275fdda9b5919c70020192c58dfdf373df Mon Sep 17 00:00:00 2001
From: Hideki Yamane <henrich@debian.org>
Date: Sun, 25 Feb 2018 19:15:46 +0900
Subject: [PATCH] use force_md5 for breezy dapper edgy

Oldest Ubuntu releases needs to be specified to use force_md5
---
 scripts/breezy | 1 +
 scripts/dapper | 1 +
 scripts/edgy   | 1 +
 3 files changed, 3 insertions(+)

diff --git a/scripts/breezy b/scripts/breezy
index 2148cec..f15967a 100644
--- a/scripts/breezy
+++ b/scripts/breezy
@@ -3,6 +3,7 @@ mirror_style release
 download_style apt
 finddebs_style from-indices
 variants - buildd
+force_md5
 
 case $ARCH in
   alpha|ia64) LIBC="libc6.1" ;;
diff --git a/scripts/dapper b/scripts/dapper
index af83c0c..b1e44d0 100644
--- a/scripts/dapper
+++ b/scripts/dapper
@@ -10,6 +10,7 @@ mirror_style release
 download_style apt
 finddebs_style from-indices
 variants - buildd
+force_md5
 
 case $ARCH in
   alpha|ia64) LIBC="libc6.1" ;;
diff --git a/scripts/edgy b/scripts/edgy
index 01d6568..9f0dda4 100644
--- a/scripts/edgy
+++ b/scripts/edgy
@@ -10,6 +10,7 @@ mirror_style release
 download_style apt
 finddebs_style from-indices
 variants - buildd fakechroot
+force_md5
 
 if doing_variant fakechroot; then
     test "$FAKECHROOT" = "true" || error 1 FAKECHROOTREQ "This variant requires fakechroot environment to be started"
-- 
2.16.2

